database/seeder.go

```diff
@@ -0,0 +1,245 @@
+package database
+
+import (
+	"fmt"
+	"log"
+	"reflect"
+	"time"
+
+	"github.com/gookit/color"
+	"github.com/goravel/framework/contracts/console"
+	"github.com/goravel/framework/contracts/database"
+)
+
+type Seeder struct {
+	Seeders []database.Seeder
+	Command console.Context
+	Called  []string
+}
+
+func NewSeeder() database.Seeder {
+	return &Seeder{}
+}
+
+func (s *Seeder) Run(ctx console.Context) error {
+	// Implement the seeder logic here
+	return nil
+}
+
+func (s *Seeder) Register(seeders []database.Seeder) {
+	s.Seeders = append(s.Seeders, seeders...)
+}
+
+func (s *Seeder) GetSeeder(name string) database.Seeder {
+	var seeder database.Seeder
+	for _, item := range s.Seeders {
+		itemType := reflect.TypeOf(item).Elem()
+		if itemType.String() == name {
+			seeder = item
+			break
+		}
+	}
+	return seeder
+}
+
+// Call executes the specified seeder(s).
+// Example usage:
+//
+//	seeder := &Seeder{}
+//	seeder.Call(&UserSeeder{}, false, nil)
+//	seeder.Call([]interface{}{&UserSeeder{}, &PostSeeder{}}, true, nil)
+//
+// Parameters:
+//   - class (interface{}): The seeder class or a slice of seeder classes to execute.
+//   - silent (bool): Determines whether the execution should be silent or display output.
+//   - parameters ([]interface{}): Optional. Additional parameters to pass to the seeder(s).
+//
+// Returns:
+//   - error: An error if the execution fails.
+func (s *Seeder) Call(class interface{}, silent bool, parameters []interface{}) error {
```

Comment:
1. The `parameters` can be passed directly if needed: `seeder.Call([]seeder.Seeder{&UserSeeder{Parameters: *}})`;
2. I think `silent` can not be supported now;
```suggestion
func (s *Seeder) Call(seeders []seeder.Seeder) error {
```

Reply:
Done